VEHICLE/N_0x8aa9180de2fedd45.md

```diff
@@ -5,11 +5,14 @@ ns: VEHICLE
 
 ```c
 // 0x8AA9180DE2FEDD45 0xAE040377
-void _0x8AA9180DE2FEDD45(Vehicle vehicle, BOOL p1);
+void _0x8AA9180DE2FEDD45(Vehicle vehicle, BOOL enableobstacleavoidance);
+```
+```
+Will disable a plane or a helicopter's need to swerve around object in its heightmap.  Will ensure plane flys directly to it's destination or die trying!
```

Comment:
Not sure why you're putting the description inside code blocks, code blocks are reserved for native imports, therefore the description should be outside the code blocks. In this case just remove the code blocks wrapping the description.

Reply:
Sorry about that! This has been taken care of! 
Thank you for that.  It may still have errors, so please let me know if there are other things that should be changed! 

Thanks! 😄